R/prep_data.R

```diff
@@ -283,27 +300,25 @@ prep_data <- function(d,
            list_variables(removing))
 
     # Convert date columns to useful features and remove original. ------------
-    if (!is.logical(convert_dates)) {
-      if (!is.character(convert_dates))  #  || is.null(names(convert_date))
-        stop("convert_dates must be logical or features for step_date")
-      sdf <- convert_dates
-      convert_dates <- TRUE
-    }
-    if (convert_dates) {
-      # If user didn't provide features, set them to defaults
-      if (!exists("sdf"))
-        sdf <- c("dow", "month", "year")
+    if (!is.character(convert_dates))
```

Comment:
Most of prep data works with logical as well as character. Can you change this logic to have `convert_dates = TRUE` go to the default and `convert_dates = FALSE` go to none?

That with a quick test and you're done on this PR. Great job!